src/Pact/Analyze/Check.hs

```diff
@@ -437,6 +445,46 @@ moduleTables modules ModuleData{..} = do
 
     pure $ Table tabName schema invariants
 
+moduleCapabilities :: ModuleData -> ExceptT VerificationFailure IO [Capability]
+moduleCapabilities md = do
+    toplevels <- withExceptT ModuleCheckFailure $ ExceptT $
+                   fmap sequence $ traverse typecheck defcapRefs
```

Comment:
I left off the `ExceptT` in `typecheck` because 2 out of the 3 clients are non-transformer code, and we only need to add the `ExceptT` wrapper for the 1 client that is lives in a transformer.

But this is a great catch that if `typecheck` _did_ return an `ExceptT` in this case, it removes the need for `fmap sequence` -- I am going to change:

`withExceptT ModuleCheckFailure $ ExceptT $ fmap sequence $ traverse typecheck defcapRefs`

to

`withExceptT ModuleCheckFailure $ traverse (ExceptT . typecheck) defcapRefs`